contracts/AaveV3YieldSource.sol

```diff
@@ -43,7 +43,7 @@ contract AaveV3YieldSource is ERC20, IYieldSource, Manageable, ReentrancyGuard {
     string name,
     string symbol,
     uint8 decimals,
-    address owner
+    address indexed owner
```

Comment:
hmm...seems silly to emit this as there is already an OwnershipTransferred event.

no big deal